x/mint/README.md

```diff
@@ -0,0 +1,33 @@
+# `x/mint`
+
+`x/mint` is a fork of the Cosmos SDK `x/mint` module that makes some changes to the inflation mechanism. See ADR-019 for the motivation behind these changes.
```

Comment:
I think it would be better if there was a brief description to what these changes are. Most people looking for information around minting will come here first (unless we want to house this information under the `spec` directory)

Reply:
Good idea, will list the changes here.